domains/service/src/system_domain.rs

```diff
@@ -536,32 +534,6 @@ where
         Box::pin(system_domain_listener),
     );
 
-    spawn_essential.spawn_blocking("system-domain-transaction-gossip", None, {
```

Comment:
Instead of just the logic of gossiping transactions, the whole `cross-domain-message-gossip` and everything related should be removed as well.